docs/logs/getting-started-console/README.md

```diff
@@ -31,34 +31,41 @@ package:
 dotnet add package OpenTelemetry.Exporter.Console
 ```
 
-Update the `Program.cs` file with the code from [Program.cs](./Program.cs):
+Copy the [FoodSupplyLogs.cs](./FoodSupplyLogs.cs) and [Program.cs](./Program.cs)
+files to the project folder.
 
 Run the application again (using `dotnet run`) and you should see the log output
 on the console.
 
 ```text
-LogRecord.Timestamp:               2023-01-21T00:33:08.1467491Z
-LogRecord.CategoryName:            GettingStarted.Program
-LogRecord.LogLevel:                Information
-LogRecord.State (Key:Value):
-    name: tomato
-    price: 2.99
-    OriginalFormat (a.k.a Body): Hello from {name} {price}.
-LogRecord.EventId:                 123
+LogRecord.Timestamp:               2023-08-03T20:54:42.1933237Z
+LogRecord.CategoryName:            SourceGeneration.Program
+LogRecord.Severity:                Info
+LogRecord.SeverityText:            Information
+LogRecord.FormattedMessage:        Food `artichoke` price changed to `9.99`.
+LogRecord.Body:                    Food `{name}` price changed to `{price}`.
+LogRecord.Attributes (Key:Value):
+    name: artichoke
+    price: 9.99
+    OriginalFormat (a.k.a Body): Food `{name}` price changed to `{price}`.
+LogRecord.EventId:                 1
+LogRecord.EventName:               FoodPriceChanged
```

Comment:
Add Console output for logging `FoodRecallNotice` as well.

Reply:
I think we could simply past the output from an actual run which includes Resource information as well.

Reply:
Would it be too long in your opinion?

Reply:
I've included the full output, just took a look at the preview and it seems okay.

Reply:
It would be long but it's assuring for a beginner to not see any discrepancy in their console output compared to what we tell them here.

Or we could just add a single line after the `FoodPriceChanged` saying:

`Omitted remaining output for brevity."